docker-compose.yml

```diff
@@ -8,20 +8,21 @@ services:
       - './images:/var/www/images.nypl.org'
       - './cache:/ifs/prod/iiif-imagecache'
       - './repo:/ifs/prod/repo'
+      - './app:/usr/src/app'
     ports:
       - '8182:8182'
 
-  shim:
-    build: ./nginx-configs
-    ports:
-      - "8080:80"
-  db:
-    image: mysql
-    command: --default-authentication-plugin=mysql_native_password
-    environment:
-      MYSQL_ROOT_PASSWORD: password
-    volumes:
-      - ./db/data:/var/lib/mysql
-      - ./db/setup:/docker-entrypoint-initdb.d
-    ports:
-      - 3306:3306
+  # shim:
+  #   build: ./nginx-configs
+  #   ports:
+  #     - "8080:80"
+  # db:
+  #   image: mysql
+  #   command: --default-authentication-plugin=mysql_native_password
+  #   environment:
+  #     MYSQL_ROOT_PASSWORD: password
+  #   volumes:
+  #     - ./db/data:/var/lib/mysql
+  #     - ./db/setup:/docker-entrypoint-initdb.d
+  #   ports:
+  #     - 3306:3306
```

Comment:
Did you mean to keep the shim commented out?

Reply:
I should probably revert that

Reply:
done!